servers/zms/src/main/java/com/yahoo/athenz/zms/DBService.java

```diff
@@ -439,15 +444,15 @@ boolean processPolicy(ObjectStoreConnection con, Policy originalPolicy, String d
 
             if (!ignoreDeletes) {
                 for (Assertion assertion : delAssertions) {
-                    if (!con.deleteAssertion(domainName, policyName, assertion.getId())) {
+                    if (!con.deleteAssertion(domainName, policyName, null, assertion.getId())) {
                         return false;
                     }
                 }
                 auditLogAssertions(auditDetails, "deleted-assertions", delAssertions);
             }
 
             for (Assertion assertion : addAssertions) {
-                if (!con.insertAssertion(domainName, policyName, assertion)) {
+                if (!con.insertAssertion(domainName, policyName, null, assertion)) {
```

Comment:
Same as in previous comment, the function processPolicy is only called from APIs that use the active version only.
However, to prevent bugs in case the code changes in the future and this function will be used for both active and disable policy versions, I'll change it as requested.